sled-agent/src/bootstrap/early_networking.rs

```diff
@@ -804,6 +829,7 @@ impl RackNetworkConfigV0 {
                 .map(|uplink| PortConfigV1::from(uplink))
                 .collect(),
             bgp: vec![],
+            bfd: vec![],
```

Comment:
Is this actually good enough? Presumably we are already on `v1`. Will we serialize an existing `v1` into an empty array if the field isn't present?

Reply:
This comment really should have been on the `v1` type addition. But the question still stands. Do we need a v2?

Reply:
I've put a `#[serde(default)]` on the new field, so i'm trying to avoid a v2 if possible - but i'm not sure which way it's going to go yet

Reply:
Got it. Thanks!